### PR TITLE
contributing fixes

### PR DESCRIPTION
**Pull request by IvanKarpan**
**What I did:**
Fixed a problem with dev server script and updated CONTRIBUTING.md

**Why I did it:**
Wanted to poke around the application and learned the documentation was out of sync with how dev environment seemed to be set up

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,26 +11,20 @@ In your copy of the **dauntless-builder** directory, use the following in your s
...
-
 # Install/update all dependencies via
 yarn
 
-# You have to bootstrap stuff once, this will create symlinks to make stuff easier
-yarn bootstrap
-
 # And then finally build the assets with
-yarn build
+yarn build-dev
```
Comment (atomicptr) [inline CONTRIBUTING.md:21]: The only problem with this is that build-dev doesn't build the data :)

Comment (IvanKarpan) [reply CONTRIBUTING.md:21]: Pushed an update

**Comment by atomicptr** [comment]
Yeah looks good now, tyvm for the contribution!

**Comment by IvanKarpan** [comment]
My pleasure, thank you for the tool :)

**Pull request merged**